Upstream have introduced support for QUIC, but are using an in-tree snapshot of
a repo without any releases: while we determine how we want to handle this
scenario, just build Samba without QUIC support entirely.

--- a/third_party/ngtcp2/wscript
+++ b/third_party/ngtcp2/wscript
@@ -5,6 +5,8 @@ from waflib import Logs
 from samba_utils import save_file
 
 def configure(conf):
+    conf.SET_TARGET_TYPE('ngtcp2', 'EMPTY')
+    return
     if not sys.platform.startswith('linux'):
         Logs.warn('libngtcp2 requires linux')
         conf.SET_TARGET_TYPE('ngtcp2', 'EMPTY')
diff --git a/third_party/quic/wscript b/third_party/quic/wscript
index 6ad6096..5705679 100644
--- a/third_party/quic/wscript
+++ b/third_party/quic/wscript
@@ -4,6 +4,8 @@ import sys
 from waflib import Logs
 
 def configure(conf):
+    conf.SET_TARGET_TYPE('quic', 'EMPTY')
+    return
     if not sys.platform.startswith('linux'):
         Logs.warn('libquic requires linux')
         conf.SET_TARGET_TYPE('quic', 'EMPTY')
